playbooks/allocate_pubcloud.yml

```diff
@@ -58,7 +64,7 @@
           {% for instance in rax.success %}
           {{instance.name}} ansible_host={{instance.accessIPv4}} ansible_user=root
           {% endfor %}
-        dest: '{{lookup("env", "WORKSPACE")}}/rpc-gating/playbooks/inventory/hosts'
+        dest: '{{inventory_path}}/hosts'
```

Comment:
I missed that the playbook was localhost targeted, and see that now. That still doesn't help me understand though.

I think I'm getting it - you need a var that's overridable here. So it's not really about changes to this playbook, but instead about how the playbook is used.